data/json/npcs/exodii/exodii_merchant_missions.json

```diff
@@ -242,5 +265,199 @@
+        }
+      }
+    },
+    "responses": [
+      {
+        "text": "I think I got most of that.  A metal and concrete building with some kind of metal pole on top that you want me to fix?",
+        "topic": "TALK_EXODII_MERCHANT_mission_2_intro1"
+      },
+      { "text": "What?  A jut?  Fig and a fiddle?", "topic": "TALK_EXODII_MERCHANT_clarify" },
+      { "text": "Can we talk about something else?", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "I've gotta go.  See you later, Rubik.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_mission_2_clarify",
+    "type": "talk_topic",
+    "dynamic_line": "A jut be a long, thin post, kennit.  The store-cabin be made o' con-crete an' rusty iron, and atop it be a glintin' post.  The post be broken, I'd razz.  Turnt off.  Rubik's got a tool for you to pop, make it turnt on.",
+    "//~": "A jut is a long, thin pole.  The warehouse is made of concrete and rusty iron, and atop it there's a glistening pole.  The pole is broken, turned off.  I got a tool for you to turn it back on.",
+    "responses": [
+      {
+        "text": "OK, got it.  A metal and concrete building with some kind of metal pole on top that you want me to fix",
+        "topic": "TALK_EXODII_MERCHANT_mission_2_intro1"
+      },
+      { "text": "Can we talk about something else?", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "I've gotta go.  See you later, Rubik.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_mission_2_intro1",
+    "type": "talk_topic",
+    "dynamic_line": "Just so.  'Tis like yarkin' fit f'r me beloved jaryi, y'kennit so fine.  What say you?",
+    "//~": "Exactly.  This is some talking fit for my beloved jaryi.  What do you think?",
+    "responses": [
+      {
+        "text": "What, now that I think on it, I am not sure what you meant about juts and figs.",
+        "topic": "TALK_EXODII_MERCHANT_mission_2_clarify"
+      },
+      { "text": "Shouldn't your quads be able to do that?", "topic": "TALK_EXODII_MERCHANT_mission_2_explain" },
+      { "text": "Any clues on how to find it?", "topic": "TALK_EXODII_MERCHANT_mission_2_find" },
+      { "text": "What's in this for me?", "topic": "TALK_EXODII_MERCHANT_mission_2_reward" },
+      {
+        "text": "Then I'll find it.",
+        "topic": "TALK_DONE",
+        "effect": { "assign_mission": "EXODII_MISSION_WAREHOUSE" }
+      },
+      {
+        "text": "Beloved jaryi?  What did you just call me?",
+        "topic": "TALK_EXODII_MERCHANT_jaryi",
+        "arithmetic": [ { "global_val": "var", "var_name": "u_know_jaryi" }, "=", { "const": 1 } ]
```

Comment:
Not sure what this is supposed to be.

Reply:
Variable tracking that can be used to add more conversation topics about where Rubik is from. I'm not a huge fan of using arithmetic for that stuff myself

Reply:
Sorry I wasn't clear in my comment there's an error there saying that arithmetic can't be used that way.

Reply:
I fixed that line. Thanks for pointing it to me